napari/layers/base/base.py

```diff
@@ -1020,6 +1020,13 @@ def _update_draw(self, scale_factor, corner_pixels, shape_threshold):
     @property
     def displayed_coordinates(self):
         """list: List of currently displayed coordinates."""
+        warnings.warn(
+            "displayed_coordinates is deprecated and will be removed in version 0.4.8."
+            " It should no longer be used as layers should will soon not know"
+            " which dimensions are displayed.",
```

Comment:
@sofroniewn in the interim, is there a way for users to discover this information in a more future-proof way?

Reply:
Yes, I've updated that

Reply:
Actually @jni this is a little more complicated as we'll deprecate `layer.coordinates` too, probably in this release too, as we don't want to store cursor information on the layer, see #2249. My recommendation is we merge with this warning now and then update before the next release?